docs/howto/custom-management-commands.txt

```diff
@@ -191,6 +191,17 @@ Attributes
 All attributes can be set in your derived class and can be used in
 :class:`BaseCommand`’s :ref:`subclasses<ref-basecommand-subclasses>`.
 
+.. attribute:: BaseCommand.suppressed_base_arguments
+
+    A set object with the default options entries, which are suppressed.
+    Use the option names you want to suppress the help output for, eg
+    '--verbosity'.
+    Example: ``manage.py runserver`` command doesn't support both --verbosity
```

Comment:
> For example, `manage.py runserver` doesn't support

Also, use backticks around the option names.

Reply:
I've noticed the format is: ``:option:`--version` ``, so using this syntax.